### ISSUE TITLE
Not a typed array

### ISSUE THREAD

**Issue by zgoda** [websockets/wscat#39]
I just installed `wscat` but:

```
$ wscat -c ws://localhost:8080/ws
/usr/local/lib/node_modules/wscat/node_modules/ws/lib/PerMessageDeflate.js:8
const TRAILER = Buffer.from([0x00, 0x00, 0xff, 0xff]);
                       ^

TypeError: this is not a typed array.
    at Function.from (native)
    at Object.<anonymous> (/usr/local/lib/node_modules/wscat/node_modules/ws/lib/PerMessageDeflate.js:8:24)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/wscat/node_modules/ws/lib/WebSocket.js:16:27)
    at Module._compile (module.js:410:26)
```

All invocations of `wscat` ends this way.

IDK if this is of any importance:

```
$ npm version
{ npm: '3.5.2',
  ares: '1.10.1-DEV',
  http_parser: '2.5.0',
  icu: '55.1',
  modules: '46',
  node: '4.2.6',
  openssl: '1.0.2g-fips',
  uv: '1.8.0',
  v8: '4.5.103.35',
  zlib: '1.2.8' }
```

**Comment by lpinca** [comment]
Yes, the latest version of `wscat` is using a `ws` [version](https://github.com/websockets/wscat/blob/c5a99e8b287a6cde99b7e73cf6aa3382e0dfcfb6/package.json#L24) which only works on Node.js >= 4.5.

**Comment by lpinca** [comment]
I've just released 2.0.1 which should fix this issue. Thanks for the report.
